### PR TITLE
ci: configure Dependabot version updates

### PR DESCRIPTION
**Pull request by kirillmorozov**
Configure Dependabot for daily updates to container images in Dockerfiles.

Related-to: k3s-io/k3s#7440

### CODE REVIEW

**Comment by kirillmorozov** [comment]
Please let me know if labels and reviewers also need to be configured.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+version: 2
+ updates:
+   - package-ecosystem: "docker"
+     directory: "/"
+     schedule:
+       interval: "daily"
```
Comment (dereknola) [inline .github/dependabot.yml:7]:
```suggestion
      reviewers:
         - "k3s-io/k3s-dev"
     schedule:
       interval: "monthly"
```

**Review by dereknola — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (dereknola) [reply .github/dependabot.yml:7]: Yeah you will want to setup the devs to be added as reviewers. Additionally, for these smaller repos, reducing the turnover is nicer so we don't get swamped by a bunch of tiny PRs.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+version: 2
+ updates:
+   - package-ecosystem: "docker"
+     directory: "/"
+      reviewers:
+         - "k3s-io/k3s-dev"
```
Comment (dereknola) [inline .github/dependabot.yml:7]:
```suggestion
     reviewers:
        - "k3s-io/k3s-dev"
```

Comment (dereknola) [reply .github/dependabot.yml:7]: Sorry my suggestion was off by a space :upside_down_face:

**Review by dereknola — CHANGES_REQUESTED** [review]

**Review by dereknola — APPROVED** [review]

File: .github/dependabot.yml (hunk 2 above)
Comment (kirillmorozov) [reply .github/dependabot.yml]: Okay, I've fixed indentation and cleaned commit history.

**Review by brandond — APPROVED** [review]

**Pull request merged**